stream-transformation-test/stream-transformation-it/src/test/java/uk/gov/justice/framework/tools/replay/StreamTransformationIT.java

```diff
@@ -0,0 +1,210 @@
+                               final String eventToolJarLocation,
+                               final String standaloneDSLocation) {
+        return format("java -jar -Dorg.wildfly.swarm.mainProcessFile=%s -Devent.transformation.jar=%s %s -c %s",
+                mainProcessFilePath,
+                streamJarLocation,
+                eventToolJarLocation,
+                standaloneDSLocation);
+    }
+
+    private static DataSource initDatabase(final String dbUrlPropertyName,
+                                           final String dbUserNamePropertyName,
+                                           final String dbPasswordPropertyName,
+                                           final String... liquibaseChangeLogXmls) throws Exception {
+        final BasicDataSource dataSource = new BasicDataSource();
+
+        dataSource.setDriverClassName(TEST_PROPERTIES.value("db.driver"));
+
+        dataSource.setUrl(TEST_PROPERTIES.value(dbUrlPropertyName));
+        dataSource.setUsername(TEST_PROPERTIES.value(dbUserNamePropertyName));
+        dataSource.setPassword(TEST_PROPERTIES.value(dbPasswordPropertyName));
+        boolean dropped = false;
+        final JdbcConnection jdbcConnection = new JdbcConnection(dataSource.getConnection());
+
+        for (String liquibaseChangeLogXml : liquibaseChangeLogXmls) {
+            Liquibase liquibase = new Liquibase(liquibaseChangeLogXml,
+                    new ClassLoaderResourceAccessor(), jdbcConnection);
+            if (!dropped) {
+                liquibase.dropAll();
+                dropped = true;
+            }
+            liquibase.update("");
+        }
+        return dataSource;
+    }
+
+    public boolean eventStoreTransformedEventPresent() throws SQLException {
+        final Stream<Event> eventLogs = EVENT_LOG_JDBC_REPOSITORY.findAll();
+        final Optional<Event> event = eventLogs.filter(item -> item.getStreamId().equals(STREAM_ID)).findFirst();
+
+        return event.isPresent();
+    }
+
+    private String getResource(final String pattern) {
+        final File dir = new File(this.getClass().getClassLoader().getResource("").getPath());
+        final FileFilter fileFilter = new WildcardFileFilter(pattern);
+        return dir.listFiles(fileFilter)[0].getAbsolutePath();
+    }
+
+    public void runCommand(final String command) throws Exception {
+        final Process exec = Runtime.getRuntime().exec(command);
+        final BufferedReader reader =
+                new BufferedReader(new InputStreamReader(exec.getInputStream()));
+
+        String line = "";
+        while ((line = reader.readLine()) != null) {
+            System.out.println(line);
+        }
+    }
+
+    private void insertEventLogData() throws SQLException, InvalidSequenceIdException {
+        EVENT_LOG_JDBC_REPOSITORY.insert(eventLogFrom("people.events.person-gender-updated", 1L));
+        EVENT_STREAM_JDBC_REPOSITORY.insert(STREAM_ID ,true);
```

Comment:
i think there is a method with a single parameter, by default it sets as active.